sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -750,8 +749,8 @@ class Std {
         case (l: Val.Obj, r: Val.Obj) =>
           val kvs = for {
             k <- (l.visibleKeyNames ++ r.visibleKeyNames).distinct
-            val lValue = Option(l.valueRaw(k, l, pos)(ev))
-            val rValue = Option(r.valueRaw(k, r, pos)(ev))
+            lValue = Option(l.valueRaw(k, l, pos)(ev))
+            rValue = Option(r.valueRaw(k, r, pos)(ev))
```

Comment:
Does removing the immutability here fix a problem that you found?

Reply:
it was a intellij recommendation (val is deprecated or something like taht)